skills/skills-and-code-style-contribution/rule-template.md

```
@@ -0,0 +1,163 @@
+
+// good - O(n), and the intent reads the same
+const byId = new Map(accounts.map(account => [account.key, account]));
+```
+````
+
+## The `description:` frontmatter
+
+This line decides whether the skill gets loaded at all, so it has to say both _what_ the rules cover
+and _when_ to reach for them. Formula:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)